### PR TITLE
Add new phishing domains

### PR DESCRIPTION
**Pull request by anishshandilya**
Add new phishing domains

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -956,6 +956,50 @@
...
+    "web3-blur.com",
+    "free-blur.com",
+    "blurnft.org",
+    "register-pudgypenguin.com",
+    "blur.markets",
+    "blur-overview.com",
+    "blur.surf",
+    "blur-nft.space",
+    "aptoslive.io",
+    "moonbirds.by?r=slblkc753b",
```
Comment (ktian94) [inline src/config.json]: just moonbirds.by

File: src/config.json [hunk 2]
```diff
@@ -956,6 +956,50 @@
...
+    "moonbirdsbtc.com",
+    "moonblrdss.xyz",
+    "suidex.io",
+    "noreply@luma-mail.com",
```
Comment (ktian94) [inline src/config.json]: rm

File: src/config.json [hunk 3]
```diff
@@ -956,6 +956,50 @@
...
+    "luma-mail.com",
```
Comment (ktian94) [inline src/config.json]: rm

File: src/config.json [hunk 4]
```diff
@@ -956,6 +956,50 @@
...
+    "blure-oficial.org",
+    "ethdenver-attending.com",
+    "waitlist-doodles.app",
+    "blur-project.com",
+    "ethdenver-application.com",
+    "freesolana.app",
+    "blur-site.com",
+    "staderlabs.com",
```
Comment (ktian94) [inline src/config.json]: rm

**Review by ktian94 — CHANGES_REQUESTED** [review]

**Review by ktian94 — COMMENTED** [review]
Add [vwww-looksrare.com](https://vwww-looksrare.com/)

File: src/config.json [hunk 5]
```diff
@@ -956,6 +956,50 @@
...
+    "register-pudgypenguin.com",
+    "blur.markets",
+    "blur-overview.com",
+    "blur.surf",
+    "blur-nft.space",
+    "aptoslive.io",
+    "moonbirds.by?r=slblkc753b",
+    "moonbirdsbtc.com",
+    "moonblrdss.xyz",
+    "suidex.io",
```
Comment (ktian94) [inline src/config.json]: rm

**Review by ktian94 — APPROVED** [review]

**Review by AlexHerman1 — APPROVED** [review]
LGTM.

**Pull request merged**